tests/test_context.py

```diff
@@ -537,17 +470,7 @@ def setUpClass(cls):
         super().setUpClass()
         component.registry.register(name="simple_component", component=SimpleComponent)
 
-    def setUp(self):
-        self.patcher = patch(
```

Comment:
This way of patching is no longer needed as we achieve the same with the decorator.